### PR TITLE
Cookies and privacy policy

### PR DESCRIPTION
**Pull request by johnpeart**
This PR attempts to add two pages to the site and link them from the footer. It implements #47.
The pages are not populated with the substantive content, as we should write this after the service is built so that it is accurate.

Screenshot of the cookies page
.

### CODE REVIEW

**Comment by jonodrew** [comment]
Can you talk me through the addition of the 
```
{% block breadcrumbs %}
		 ...
{% endblock %}
```
You've added in a couple of places? I'm not sure I understand what they're for

**Comment by johnpeart** [comment]
If your question is "why are they here" rather than "why have you done it this way"; the breadcrumbs are less for this tool and more about the overall domain navigation. We could remove them if you don't like them.

What I'm trying to do with all the 'tools' we're creating for volunteers to use is have them linked from [a single page](https://civilservice.lgbt/tools) and then use the breadcrumbs to give people a sense of hierarchy.

If the question means "why have you done it this way"; then I was trying to extrapolate from what you'd done. In Liquid (which is what they use for Jekyll), you include a piece of reusable code by using {% includes nameoffile.html %}; I assumed this was the equivalent!

File: app/templates/base.html [hunk 1]
```diff
@@ -62,6 +64,16 @@ <h2 class="footer--heading">Related links</h2>
 							<a class="footer--link" href="https://www.civilservice.lgbt/mentoring" title="Find out more about our mentoring programme">
 								Our mentoring programme
 							</a>
+            </li>
+						<li class="footer--list-item">
+							<a class="footer--link" href="/privacy-and-data" title="Read the cookies policy">
```
Comment (jonodrew) [inline app/templates/base.html:69]: Should this say 'Read the privacy and data policy' ?

**Comment by jonodrew** [comment]
Ah yes I see what you were aiming for! I like it, it's good to reduce duplication. I think that's why I'm confused about the repetition, I think we should be able to define the breadcrumbs block in the base template, and then it should be automatically included when we `extend 'base.html`. Can you give that a try and see if it helps?

File: app/templates/base.html (hunk 1 above)
Comment (johnpeart) [reply app/templates/base.html:69]: Yes. Will fix.

**Comment by johnpeart** [comment]
@jonodrew I think I have a better understanding of how the `{% block %}` works now. 
I've amended the breadcrumbs in [7800f55](https://github.com/jonodrew/mentor-match/pull/51/commits/7800f5511ad171b94285cafb7d06172d0f25cc2b); let me know if this is better!

050c59c reapplies routes for the pages which for some reason disappeared when merging other commits.

File: app/templates/base.html (hunk 1 above)
Comment (johnpeart) [reply app/templates/base.html]: Fixed in fddf1b8

**Comment by johnpeart** [comment]
76c5627 should extend the use of `{% block %}` further to create a more 'DRY' header component for the page caption, title and excerpt.

**Review by jonodrew — APPROVED** [review]

**Pull request merged**